hca-api-stub/voa-rest.js

```diff
@@ -10,17 +10,17 @@ function attach(app) {
   app.set('restApiRoot', '/api/hca/v1');
 
   // TODO: use `NODE_ENV` should probably determining which endpoint is selected.
-  const endpoint = {
-    // ES "unofficial" (provided by Joshua) endpoints for development
-    c7401: 'http://vaausesrapp803.aac.va.gov:7401/voa/voaSvc',
-    e6401: 'https://vaausesrapp803.aac.va.gov:6401/voa/voaSvc',
-    // ES "official" endpoints for development
-    esSqa: 'https://vaww.esrstage1a.aac.va.gov/voa/voaSvc',
-    esDev: 'https://vaww.esrdev30.aac.va.gov:8432/voa/voaSvc',
-    // ES endpoints that require keys from PKI team and certs installed by Joshua
-    esPreprod: 'https://vaww.esrpre-prod.aac.va.gov/voa/voaSvc',
-    esProd: 'https://vaww.esr.aac.va.gov/voa/voaSvc'
-  };
+  // const endpoint = {
```

Comment:
Commented this out because it was giving me a lint error because we were not using `endpoint`. Changing that now though.